runtime/src/impls.rs

```diff
@@ -172,27 +175,21 @@ impl pallet_authority_discovery::Config for Runtime {
 }
 
 parameter_types! {
-	pub const ProposalBond: Permill = Permill::from_percent(5);
-	pub const ProposalBondMinimum: Balance = AVAIL;
+	pub const ProposalBond: Permill = Permill::from_percent(50);
+	pub const ProposalBondMinimum: Balance = 100 * AVAIL;
```

Comment:
This much ?

Reply:
Yes, basically to deter users to create treasury proposals, normal users wont be able to propose.